src/subapps/dataExplorer/PredicateSelector.tsx

```diff
@@ -88,77 +85,142 @@ export const PredicateSelector: React.FC<Props> = ({
             selectedPath: path,
           });
         } else {
-          onPredicateChange({ predicateFilter: null, selectedPath: null });
+          onPredicateChange({ predicate: null, selectedPath: null });
         }
 
         break;
       default:
-        onPredicateChange({ predicateFilter: null, selectedPath: null });
+        onPredicateChange({ predicate: null, selectedPath: null });
     }
   };
 
+  const getFormFieldValue = (fieldName: string) => {
+    return formRef.current?.getFieldValue(fieldName) ?? '';
+  };
+
+  const setFormField = (fieldName: string, fieldValue: string) => {
+    if (formRef.current) {
+      formRef.current.setFieldValue(fieldName, fieldValue);
+    }
+  };
+
+  const onReset = () => {
+    const form = formRef.current;
+    if (form) {
+      form!.resetFields();
```

Comment:
since you checked the nullish value of form, then you can remove the exclamation mark

Reply:
true. thanks! I'll update it.